cmd/app/serve.go

```diff
@@ -60,7 +60,7 @@ func newServeCmd() *cobra.Command {
 	cmd.Flags().String("log_type", "dev", "logger type to use (dev/prod)")
 	cmd.Flags().String("ca", "", "googleca | tinkca | pkcs11ca | fileca | kmsca | ephemeralca (for testing)")
 	cmd.Flags().String("aws-hsm-root-ca-path", "", "Path to root CA on disk (only used with AWS HSM)")
-	cmd.Flags().String("gcp_private_ca_parent", "", "private ca parent: /projects/<project>/locations/<location>/<name> (only used with --ca googleca)")
```

Comment:
Using the pool rather than the CA is an intentional decision, so that fulcio operators can increase QPS by simply adding more CAs to the pool. 

List is working as intended (besides the len(roots) bug) - For each CA in the pool, it grabs the CA chain, and appends it to a list of chains. 

The problem was that the v1 GetRootCertificate endpoint never properly supported lists of chains, so all certs would just be concatenated together. As long as you had just one CA per pool, this was fine. For v2, [TrustBundle](https://github.com/sigstore/fulcio/blob/main/fulcio.proto#L195) returns a list of list of certificates, so it works well with the pool concept.

Reply:
Thanks for the background, I changed the flag description back to what it was. I also kept the support for specifying certificateAuthority, so users can choose either option now.